stubs/python-urllib2/README.md

```diff
@@ -0,0 +1,98 @@
+
+# Dependencies
+
+ * This stub has been tested on OS X 10.11.5 and Python 2.7.10
+ * [urllib2](https://docs.python.org/2/library/urllib2.html) is part of [python 2.7 standard library](https://docs.python.org/2/library/index.html)
+
+
+# Examples
+
+## Running badssl tests with TryTLS runner.
+
+```
+% <path-to-your-trytls>/trytls -t showrunner.bundles.https.badssl_tests python run.py
+PASS badssl(True, 'sha1-2016')
+PASS badssl(False, 'expired')
+```
+
+## Running manually.
+
+```
+% python python-urllib2/run.py sha1-2016.badssl.com 443
+VERIFY SUCCESS
+
+% python python-urllib2/run.py expired.badssl.com 443
+VERIFY FAILURE
+```
+
+# Dealing with unexpected behaviour
+
+This stub tries to avoid a situation where traceback gets hidden on unexpected behaviour.
+
+Manual / example test, connecting to non-existent domain.
+```
+python python-urllib2/run.py https://www.nosuch.example.local/ 443 || echo "$?"
```

Comment:
Fixed example to match the description in: 6a26152c4d3feb9351c094ec38f134bd36805c13